lms/templates/dashboard/_dashboard_xseries_info.html

```diff
@@ -0,0 +1,34 @@
+<%page args="course_program_info, enrollment_mode" />
+<%!
+    from django.utils.translation import ugettext as _
+%>
+<%namespace name='static' file='../static_content.html'/>
+<div class="message message-status is-shown credit-message">
+    <div class="xseries-action">
+        <div class="message-copy xseries-msg">
+            <p>
+                <b class="message-copy-bold">${_("XSeries Program: Interested in more courses in this subject?")}</b>
+            <p>
+            <p class="message-copy">
+                ${_("This course is 1 of {course_count} courses in the {link_start}{course_display}{link_end}.").format(
+                    course_count=course_program_info['course_count'],
+                    link_start='<a href="{}">'.format(course_program_info['program_marketing_url']),
+                    link_end='</a>',
+                    course_display=course_program_info['display_name'],
+                )}
+            </p>
+
+        </div>
+        <%
+            xseries_btn_class = "xseries-border-btn"
+            if enrollment_mode == "verified":
+                xseries_btn_class = "xseries-base-btn";
+        %>
+        <a class="btn ${xseries_btn_class}" href="${course_program_info['program_marketing_url']}" target="_blank">
```

Comment:
@marcotuts this link is going to open in a new browser window, is that what we want?